15-7-1_and_15-c-1/15-7-1_and_15-c-1_checks.Rmd

```diff
@@ -0,0 +1,117 @@
+---
+title: "15-7-1 and 15-c-1 checks"
+author: "Katie Uzzell"
+date: "22/12/2022"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+```{r load-packages, include=FALSE}
+library(tidyr)
+library(dplyr)
+library(ggplot2)
+library(kableExtra)
+```
+
+```{r include= FALSE}
+source('15-7-1_15-c-1_example_config.R')
```

Comment:
No need to source this - the correct file will be sourced through the if statement we added for the test run. Sourcing this here hard codes it and leads to false information in the paths and messages in the report (as it always reads from the test data).

Reply:
Done